### DISCUSSION TITLE
Home Assistant in LXC - config path?

### DISCUSSION THREAD

**Discussion by alve89** [tteck/Proxmox#2865] [Help Needed] [answered]
Hi guys,
where is the config path in the new installed HA LXC container?

I thought of `/root/hass_config` but it is empty:
```
root@hassk4:~/hass_config# ls -al
total 8
drwxr-xr-x 2 root root 4096 Apr 26 19:31 .
drwx------ 5 root root 4096 Apr 26 19:57 ..
```

I'd like to put my existing installation there.

**Comment by tteck** [comment]
Home Assistant Core is `/root/.homeassistant`

**Reply by tteck** [reply, marked as answer]
Home Assistant Container is `/var/lib/docker/volumes/hass_config/_data`

**Comment by alve89** [comment]
> Home Assistant Core is `/root/.homeassistant`

```
root@hassk4:~# cd /root/.homeassistant
-bash: cd: /root/.homeassistant: No such file or directory

root@hassk4:/var/lib/docker/volumes/hass_config/_data# docker exec -it homeassistant bash
hassk4:/config# cd /root/.homeassistant
bash: cd: /root/.homeassistant: No such file or directory
```
So the fist path is confusing me...?


> Home Assistant Container is `/var/lib/docker/volumes/hass_config/_data`

But this is what I was searching for. Thank you!
